gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationStatementMapper.java

```diff
@@ -77,8 +79,13 @@ public MedicationStatement mapToMedicationStatement(RCMRMT030101UKEhrExtract ehr
             String ehrSupplyAuthoriseId = ehrSupplyAuthoriseIdExtract.get();
             MedicationStatement medicationStatement1 = new MedicationStatement();
 
+            final Meta meta = confidentialityService.createMetaAndAddSecurityIfConfidentialityCodesPresent(
+                MEDICATION_STATEMENT_URL,
+                medicationStatement.getConfidentialityCode()
+            );
+
             medicationStatement1.setId(ehrSupplyAuthoriseId + MS_SUFFIX);
```

Comment:
I think there should possibly be a better name rather than medicationStatement1 ?